Doc/whatsnew/3.14.rst

```diff
@@ -829,6 +829,9 @@ mimetypes
 
   (Contributed by Hugo van Kemenade in :gh:`129965`.)
 
+* Add :rfc:`9512` ``application/yaml`` MIME type for YAML files (``.yaml``
+  and ``.yml``). (Contributed by Kristinita in :gh:`132056`.)
```

Comment:
@Kristinita We can put your full name here if you like.

Reply:
@hugovk, thanks for adding the function `test_guess_file_type()`!

My contribution here is insignificant, and I think it would be more fair if you will add your name instead of my.

(But if the name of the author of the pull request is required, perhaps, `Sasha “Nelie” Chernykh` instead of `Kristinita` would be better).

Thanks.

Reply:
You contribution is not insignificant. I've put both our names there, thanks again!